internal/api/resolver_mutation_performer.go

```diff
@@ -317,7 +329,12 @@ func (r *mutationResolver) BulkPerformerUpdate(ctx context.Context, input BulkPe
 	updatedPerformer.Ethnicity = translator.optionalString(input.Ethnicity, "ethnicity")
 	updatedPerformer.Country = translator.optionalString(input.Country, "country")
 	updatedPerformer.EyeColor = translator.optionalString(input.EyeColor, "eye_color")
-	updatedPerformer.Height = translator.optionalString(input.Height, "height")
+	// prefer height_cm over height
+	updatedPerformer.Height, err = translator.optionalIntFromString(input.Height, "height")
+	if err != nil {
+		return nil, err
+	}
+	updatedPerformer.Height = translator.optionalInt(input.HeightCm, "height_cm")
```

Comment:
Haven't tested yet, this looks a little weird though.
Shouldnt lline 337 be first and the other code executed only if the height_cm translates to an empty value?
As it is now
- if the input has both fields and the `input.Height` is not valid we will error out and return even  if height_cm is ok and should be updated?
- Height gets overwritten every time so the first part (333-336) doesnt matter unless it errors out. Even if the `return` is removed it doesnt make sense ?

Reply:
Thanks, fixed.